beta-src/src/components/ui/WDGameVoteButtons.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+import { useState } from "react";
+import Grid from "@mui/material/Grid";
+import GameVote from "../../enums/GameVote";
+import WDButton from "./WDButton";
+import WDCheckmarkIcon from "../svgr-components/WDCheckmarkIcon";
+
+const WDGameVoteButtons = function (): React.ReactElement {
+  const [drawSelected, setDrawSelected] = useState(false);
+  const [pauseSelected, setPauseSelected] = useState(false);
+  const [cancelSelected, setCancelSelected] = useState(false);
+
+  return (
+    <Grid container spacing={1}>
+      <Grid item xs={8}>
+        <WDButton
+          color={drawSelected ? "secondary" : "primary"}
+          onClick={() => setDrawSelected(!drawSelected)}
+          startIcon={drawSelected && <WDCheckmarkIcon />}
+        >
+          <span>{GameVote.DRAW}</span>
+        </WDButton>
+      </Grid>
+      <Grid item xs={8}>
+        <WDButton
+          color={pauseSelected ? "secondary" : "primary"}
+          onClick={() => setPauseSelected(!pauseSelected)}
+          startIcon={pauseSelected && <WDCheckmarkIcon />}
+        >
+          <span>{GameVote.PAUSE}</span>
+        </WDButton>
+      </Grid>
+      <Grid item xs={8}>
+        <WDButton
+          color={cancelSelected ? "secondary" : "primary"}
+          onClick={() => setCancelSelected(!cancelSelected)}
+          startIcon={cancelSelected && <WDCheckmarkIcon />}
+        >
+          <span>{GameVote.CANCEL}</span>
+        </WDButton>
```

Comment:
we should make a local component, inside this file but outside this component that takes a GameVote and returns the WDButton with the right callbacks and text. we are repeating this pattern three times in here.